.idea/.name

```diff
@@ -1 +1 @@
-spine-base
\ No newline at end of file
+spine
```

Comment:
I think the change in this file should be reverted, as we have renamed the root project to `spine-base` intentionally.